src/common/options.cc

```diff
@@ -8506,6 +8506,20 @@ std::vector<Option> get_mds_client_options() {
     .set_description("pass default_permisions to FUSE on mount")
     .set_flag(Option::FLAG_STARTUP),
 
+#ifdef CEPH_HAVE_SPLICE
```

Comment:
I don't think this `ifdef` is necessary. It's better to keep the configs in case the clients are compiled differently.

Reply:
Okay, we can get rid of this macro, and the libfuse will be compatible it and fall back to normal data copy.